docs/zkapps/index.mdx

```diff
@@ -1,6 +1,15 @@
 ---
 title: Overview
 hide_title: true
+description: Get an understanding of zkApps, the smart contracts of Mina Protocol, powered by zero-knowledge proofs and off-chain execution. The page also provides a quickstart guide for deploying zkApps and additional resources for learning TypeScript.
```

Comment:
```suggestion
description: Get an understanding of zkApps, the smart contracts of Mina Protocol. zkApps are powered by zero knowledge proofs and off-chain execution. A quickstart guide to deploy zkApps and resources for learning TypeScript.
```

Reply:
we'll update our page frontmatter metadata after the docs are restructured (ongoing now)

so good!